### PR TITLE
Ability to whitelist file

### PR DESCRIPTION
**Pull request by dtaylor7**
[DIV-4681 - update document handler to filter files by name](https://tools.hmcts.net/jira/browse/DIV-4681)

Adds the ability to white list files by name.

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 module.exports = {
   defaultArgs: {
     uri: '/document-download/:documentName',
-    documentNamePath: 'value.DocumentFileName'
+    documentNamePath: 'value.DocumentFileName',
+    documentWhiteList: []
```
Comment (hosainnet) [inline config.js:5]: Just a minor nitpick, `documentWhitelist` is valid too

**Review by hosainnet — APPROVED** [review]
👍

File: config.js (hunk 1 above)
Comment (dtaylor7) [reply config.js:5]: what do you mean?

Comment (hosainnet) [reply config.js:5]: I think I'm used to `whitelist` as one word so no need to camelcase the L there but I'm not 100% sure which is more accurate so happy with it either way 👍

**Pull request merged**